### ISSUE TITLE
az bot prepare-deploy does not create 2 web.configs as noted.

### ISSUE THREAD

**Issue by TheFrogDaddy** [MicrosoftDocs/bot-docs#1708]
When running command `az bot prepare-deploy --code-dir "." --lang Typescript` it only creates a web.config in the root and not in the src.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 096d934c-2856-e868-efeb-6622e2a5eafb
* Version Independent ID: 8bf74926-6a35-5753-0a1a-6663ba5965b2
* Content: [Deploy your bot - Bot Service - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-deploy-az-cli?view=azure-bot-service-4.0&tabs=typescript#feedback)
* Content Source: [articles/bot-builder-deploy-az-cli.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/bot-builder-deploy-az-cli.md)
* Service: **bot-service**
* GitHub Login: @ivorb
* Microsoft Alias: **kamrani**

**Comment by zxyanliu** [comment]
Hi @TheFrogDaddy Thanks for your report. I tested it today and you are right, after running `az bot prepare-deploy --code-dir "." --lang Typescript`, only one `web.config` file will be generated in the bot's project folder, not in the `src`. However, with only one `web.config` file generated, the deployment process failed when testing in Web Chat. The deployment only succeeded after I copied and pasted the `web.config` file to the `src` folder, and run the command to deploy the bot again. 

I am not sure if the current flow to succeed is by design. It might be a bug and we need to coordinate with our engineers for a proper solution. 

Thanks again for your feedback.
